### PR TITLE
feat: remove form-input-message-group and introduce textarea storybook

### PR DESCRIPTION
**Pull request by droshev**
Introduce textarea storybook and remove class `fd-form-input-message-group`

### BREAKING CHANGES
- removed class `fd-form-input-message-group`

### CODE REVIEW

File: src/form-input-message-group.scss [hunk 1]
```diff
@@ -7,7 +7,8 @@
 $block: #{$fd-namespace}-form-input-message-group;
 
 .#{$block} {
-  display: flex;
-  flex-grow: 1;
-  position: relative;
+  // display: flex;
+  // flex-grow: 1;
+  // position: relative;
```
Comment (InnaAtanasova) [inline src/form-input-message-group.scss:12]: If not used just remove them

Comment (droshev) [reply src/form-input-message-group.scss:12]: this was just to test it

File: .storybook/preview-head.html [hunk 2]
```diff
@@ -3,146 +3,145 @@
...
-        var el = els[i];
-        el.addEventListener('click', function () {
-            var targetId = this.getAttribute("aria-controls");
-            var target = document.getElementById(targetId);
-            //dropdown
-            var isDropdown = this.getAttribute("aria-haspopup") === "true";
-            if (isDropdown) {
-                // Get all of elements with same aria controls ID
-                const elements = document.querySelectorAll('[aria-controls="' + targetId + '"]');
+    document.addEventListener('DOMContentLoaded', () => {
```
Comment (stefanoScalzo) [inline .storybook/preview-head.html:6]: this should not be part of this PR

Comment (droshev) [reply .storybook/preview-head.html:6]: otherwise the tests are not working
